x-pack/tsconfig.json

```diff
@@ -10,6 +10,7 @@
     "test/**/*"
   ],
   "compilerOptions": {
+    "resolveJsonModule": true,
```

Comment:
Initially added this to `kibana/tsconfig.json` but it caused problems in `kbn-pm/tsconfig.json` and `kibana/tsconfig.browser.json` since they do not emit `commonjs`. I can probably override this with `"resolveJsonModule": false` if necessary.

Reply:
If you wouldn't mind setting this in the root config and negating it where necessary that would keep the compilerOptions a little more consistent across projects, which is desirable.